### PR TITLE
Homepage updates for Nov 1, 2016 Hour of Code launch

### PR DESCRIPTION
**Pull request by tanyaparker**
**code.org**
image

**hourofcode.com**
image

**csedweek.org**
image

### CODE REVIEW

File: pegasus/sites.v3/code.org/views/homepage_hero.haml [hunk 1]
```diff
@@ -87,14 +87,20 @@
       - actions.each_with_index do |entry, index|
         - display = entry[:active_s] == "hide" ? "none" : "block"
         .action{style: "display: #{display}; opacity: 0"}
-          - if entry[:param1_s] == "video"
-            %img#playimage{src: "/shared/images/play-button.png", width:"40", onclick: "return showVideo_mainvideo();"}
+          - if entry[:param1_s] == "hoc_title"
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/views/homepage_hero.haml:90]: I've made the updates to the cdo-homepage gsheet, so staging is in an inbetween state until this PR is merged in and the two changes are combined together.

File: pegasus/sites.v3/csedweek.org/public/index.haml [hunk 2]
```diff
@@ -39,7 +39,7 @@ video_player: true
 
   #top
     #video.col-63.desktop-pad-right
-      = view :display_video_thumbnail, id: "cs_is_everything", video_code: "QvyTEx1wyOY", play_button: 'center', download_path: "//videos.code.org/social/cs-is-everything.mp4"
+      = view :display_video_thumbnail, id: "hoc_2016", video_code: "KsOIlDT145A", play_button: 'center'
```
Comment (tanyaparker) [inline pegasus/sites.v3/csedweek.org/public/index.haml:42]: I've asked video team for the original file so I can upload to aws. I'll add back in the download path here and in the cdo-homepages gsheet when ready.

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 3]
```diff
@@ -263,10 +263,10 @@
   map_legend_hoc_events: 'Hour of Code Event'
   map_legend_cs_tech_jam: 'Special Event'
 
-  meta_tag_og_title: 'Every child deserves opportunity'
-  meta_tag_og_description: 'The Hour of Code is a global movement reaching tens of millions of students in 180+ countries and over 45 languages. Ages 4 to 104.'
-  meta_tag_twitter_title: 'Every child deserves opportunity'
-  meta_tag_twitter_description: 'The Hour of Code is a global movement reaching tens of millions of students in 180+ countries and over 45 languages. Ages 4 to 104.'
+  meta_tag_og_title: 'The Hour of Code is coming'
+  meta_tag_og_description: 'The Hour of Code is a global movement reaching tens of millions of students of all ages in 180+ countries and over 45 languages.'
+  meta_tag_twitter_title: 'The Hour of Code is coming'
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:268]: Also updated for code.org & csedweek.org in I18n gsheet.

**Comment by breville** [comment]
lgtm

**Pull request merged**